File: 1U3/Intento2/src/Personas/Cliente.java
package Personas;

public class Cliente {
    private int id;

    private String nombre;
    private static int contadorClientes = 0;
    private String direccion;

    public Cliente( String nombre, String direccion) {
        this.id = ++contadorClientes;
        this.nombre = nombre;
        this.direccion = direccion;
    }

    public int getId() {
        return id;
    }

    public String obtenerInformacion() {
        return String.format("Id: %d Nombre: %s y direccion: %s", id, nombre, direccion);
    }

    public String getNombre() {
        return nombre;
    }

}

File: 1U3_Phyton/Cliente.py
class Cliente:
    contadorClientes = 0

    def __init__(self, nombre, direccion):
        Cliente.contadorClientes += 1
        self.id = Cliente.contadorClientes
        self.nombre = nombre
        self.direccion = direccion

    def get_id(self):
        return self.id

    def obtener_informacion(self):
        return f"Id: {self.id} Nombre: {self.nombre} y direccion: {self.direccion}"

    def get_nombre(self):
        return self.nombre
